data/hai/hai reveal 2 start.txt

```diff
@@ -0,0 +1,368 @@
+		personality plunders disables unconstrained
+		fleet "Large Core Pirates" 1
+		fleet "Small Core Pirates" 1
+
+
+mission "Hai Rescue: Meroot and Lucy"
+	landing
+	name "Return Meroot and Lucy"
+	description "Transport the scientists you rescued to <destination>. Payment is <payment>."
+	source Canyon
+	destination Newhome
+	passengers 2
+	repeat 3
+	to offer
+		"combat rating" > 50
+		has "Unwanted Cargo: done"
+		not "event: hai-human treaty signed"
+		not "Hai Rescue: Meroot and Lucy: offered"
+		"Hai Rescue: Meroot and Lucy (offer count)" < 3
+	on offer
+		"Hai Rescue: Meroot and Lucy (offer count)" ++
+		conversation
+			`As you walk through a major street, you notice a small alcove in the side of one building. Your sharp pilot eyes spot a woman within, with tufts of hair across her body. She's probably a Hai. Another woman, clearly human, is walking nearby carrying some food. She looks around to see if she's being followed before entering the alcove and covering it with a sheet of steel.`
+			`	Do you want to investigate?`
+			choice
+				`	(Investigate.)`
+				`	(Come back later.)`
+					defer
+			`	You knock on the steel sheet and a woman within the alcove slides it to the side. As she emerges, she carefully moves her body to block your view of the alcove.`
+			`	She asks nervously, "Yes, can I help you?"`
+			choice
+				`	"Why are you hiding a Hai in there?"`
+					goto hiding
+				`	"Can I help you?"`
+					goto help
+				`	"No, I was just walking by. Sorry to bother you."`
+					decline
+			label hiding
+			`	"Please don't tell anyone! We were both kidnapped, but we managed to escape. Now we're hiding here, hoping the men don't find us again."`
+			`	"Actually, if you can give us a ride to <planet>, we can pay when we get there. You'll be followed though, so I hope you have a fast ship."`
+				goto "come in"
+			label help
+			`	"Yes! Some men kidnapped my friend and I, but we managed to escape. We've been hiding here, hoping they don't find us again. If you can fly us to <planet>, we can pay you when we arrive. I hope you have a fast ship; they'll probably chase us."`
+				goto "come in"
+			label "come in"
+			`	"Come in, and we'll talk about it."`
+			choice
+				`	(Enter their alcove.)`
+				`	"No, I can't help you get home."`
+					decline
+			`	The alcove is a dead air space between two utility rooms in the basement of a warehouse. The hole in the wall looks like it was recently dug out by a laser cutter. There's a small refrigerator, a two-person cot, an air processor, and a shelf with typical household items. Books are strewn everywhere; the women must be avid readers. It seems they've set up this tiny space as a decent, albeit small, apartment.`
+			`	"I'm Lucy," says the human woman.`
+			`	"And I'm Meroot," adds the Hai woman. "I was on Newhome walking to my research lab when some men pulled me into an alley and zapped me with a tazer. I woke up in the brig of a space ship."`
```

Comment:
```suggestion
			`	"And I'm Meroot," adds the Hai woman. "I was on Newhome walking to my research lab when some men pulled me into an alley and zapped me with a taser. I woke up in the brig of a space ship."`
```
Based on [this](https://www.dailywritingtips.com/taser-or-tazer-tazing-or-tasering/#:~:text=In%20trying%20to%20find%20out,Swift's%20Electric%20Rifle.%E2%80%9D&text=I'm%20leaning%20towards%20Taser,tazed%2C%20tazing%20for%20the%20verb.), I _think_ taser should be spelled with an "s". Weird stuff.